src/reducers/typeReducer.js

```diff
@@ -0,0 +1,10 @@
+import { ON_TYPE } from '../actions/types';
+
+export default (state = '', action) => {
+    switch (action.type) {
+        case ON_TYPE:
+            return action.payload;
```

Comment:
Lets think about this one as well. What are we actually storing at the top level of our typeReducer? (we should change this to `searchTermReducer`). Do we Want to maybe define?

```
currrentSearchTerm: action.payload
```